client/my-sites/themes/theme-showcase.jsx

```diff
@@ -215,6 +266,7 @@ class ThemeShowcase extends React.Component {
 				) }
 				<div className="themes__content">
 					<QueryThemeFilters />
+					<Banner />
```

Comment:
We don't really need the `Banner` const -- let's just put `<ThemesBanner { ...bannerProps } />` right here.

Reply:
> Also the PhotoBlog ad is missing the image swap for mobile screens. It was supposed to show a different image on smaller size screens, but that's okay. Let's get this live to see if we're in the right ballpark.

@mapk I was not aware of another image to be used for mobile screens though. This in particular, changes the whole implementation since there's another variable we need to take into account. I think we need to discuss this in more detail, since we would need to specify several images as opposed to a single one.